src/lexical-elements.rst

```diff
@@ -1248,8 +1248,16 @@ extend exactly one :t:`line`.
 :t:`non-[comment]` :t:`construct`.
 
 :dp:`fls_ok0zvo9vcmzo`
-:t:`[Inner block doc]s`, :t:`[inner line doc]s`, :t:`[outer block doc]s`, and
-:t:`[outer line doc]s` are equivalent to  :t:`attribute` :c:`doc`.
+:t:`[Inner block doc]s` and :t:`[inner line doc]s` are equivalent to
+:t:`attribute` :c:`doc` ``#![doc = content]``, where content is a :t:`string
```

Comment:
How about something along the lines of

Inner block docs and inner line docs are equivalent to attribute doc of the form #![doc = content], where content is the string literal form of the comment, without leading //!, /*! and trailing */ characters.